tests/conftest.py

```diff
@@ -5,6 +5,7 @@
 
 @pytest.fixture(scope="session")
 def dsn():
-    yield os.environ.get(
-        "PGEON_TEST_DB", "postgresql://localhost:5432/postgres"
-    )
+    yield "postgres://reader:NWDMCE5xdipIjRrp@hh-pgsql-public.ebi.ac.uk:5432/pfmegrnargs"
```

Comment:
What about using this instead: https://docs.github.com/en/actions/using-containerized-services/about-service-containers? 

It will keep the current test logic the same.

Reply:
Yeah this was just for a test
But I think [services] only work on linux no ?